ci/gitlab/pipeline.py

```diff
@@ -8,22 +8,25 @@
 
 
 # GitLab settings
-gitlab_url = "https://gitlab.com"  # Replace with your GitLab URL
-api_token = os.environ.get("API_TOKEN")
-project_id = os.environ.get("CI_MERGE_REQUEST_PROJECT_ID")
-source_branch = os.environ.get("CI_MERGE_REQUEST_SOURCE_BRANCH_NAME")
+gitlab_url = os.environ.get("GITLAB_API_URL") or "https://gitlab.com"
```

Comment:
Not a huge deal at all but you can also do this:
```suggestion
gitlab_url = os.environ.get("GITLAB_API_URL", "https://gitlab.com")
```